### PR TITLE
Add documentation for broadlink remote platform

### PR DESCRIPTION
**Pull request by felipediel**
**Description:**

Documentation for the Broadlink remote control platform.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26528

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/broadlink.markdown [hunk 1]
```diff
@@ -3,20 +3,174 @@ title: "Broadlink"
 description: "Instructions on how to integrate Broadlink within Home Assistant."
 logo: broadlink.png
 ha_category:
+  - Remote
   - Switch
   - Sensor
 ha_release: 0.35
 ha_iot_class: Local Polling
 redirect_from:
+ - /components/remote.broadlink/
```
Comment (klaasnicolaas) [inline source/_components/broadlink.markdown]: Please remove the `- /components/remote.broadlink/`

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_components/broadlink.markdown (hunk 1 above)
Comment (felipediel) [reply source/_components/broadlink.markdown]: Done.

**Review by frenck — COMMENTED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Rebased PR onto latest `next`

**Comment by aegjoyce** [comment]
Excellent project! Would you consider adding some details about how and where the broadlink codes are stored - I have found the file in .storage but it would be good to know whether or not this file is suitable for user-editing; I have lots of codes already saved in scripts using the old switch integration and would much rather copy and paste codes I know work rather than going through the learning process again which seems to be prone to variability in code generation and results

**Comment by felipediel** [comment]
@aegjoyce Thanks! You can easily copy and paste your old codes into the file. Just respect JSON and make sure Home Assistant is disabled while you edit the file.

**Pull request merged**